source/connect.txt

```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to a Specific Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a specific database:
+
+- You can specify a database in the URL path when passing a connection
+  string. The database in the URL path is automatically selected unless
```

Comment:
It may be helpful to include a sentence here stating what the default database that mongosh connects to is (i.e., if I don't specify a database in my URI, what database do I connect to?)